src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -19,9 +19,11 @@ import PathwayMapper, { ICBioData } from 'pathway-mapper';
 import 'pathway-mapper/dist/base.css';
 import 'cytoscape-panzoom/cytoscape.js-panzoom.css';
 import 'cytoscape-navigator/cytoscape.js-navigator.css';
+import SampleManager from '../SampleManager';
```

Comment:
Minor but let's move this import before style imports. Maybe right after `PatientViewPageStore` import?